lib/options.nix

```diff
@@ -168,9 +168,7 @@ rec {
     mkStr =
       # TODO we should delegate rendering quoted string to `mkDefaultDesc`,
       # once we remove its special case for strings.
```

Comment:
Should the TODO stay ?

Reply:
Yes. That relates to `mkString` calling `toPretty` instead of delegating that to `mkDesc`.